discovery_child_development/notebooks/04_train_initial_model.py

```diff
@@ -0,0 +1,444 @@
+# %% [markdown]
+# # Train a baseline model and log on wandb
+#
+# In this notebook we:
+# * Perform some EDA on the (training) data - check distribution of scores and then use this to filter the data; check distributions of OpenAlex concepts and the sub-categories that we assigned
+# * Turn the data into the correct format for training classification models (ie a one-hot-encoding representation of the target)
+# * Define a baseline classifier than guesses that all new publications will be tagged with the most common combination of labels that we have in the data
+# * Log the model and key metrics on wandb
+
+# %%
+# Load libraries
+import pandas as pd
+import numpy as np
+import os
+import pickle
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import (
+    accuracy_score,
+    precision_score,
+    recall_score,
+    f1_score,
+    hamming_loss,
+    jaccard_score,
+    classification_report,
+)
+from sklearn.base import BaseEstimator, ClassifierMixin
+
+## plotting libraries
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+import wandb
+
+## nesta ds
+from nesta_ds_utils.loading_saving import S3
+
+## project code
+from discovery_child_development import PROJECT_DIR, logging
+from discovery_child_development.utils import bigquery, labelling_utils
+from discovery_child_development.utils import wandb as wb
+from discovery_child_development.utils.io import import_config
+
+bigquery.find_credentials("GOOGLE_SHEETS_CREDENTIALS")
+
+## constants
+
+S3_BUCKET = os.environ["S3_BUCKET"]
+PARAMS = import_config("config.yaml")
+CONCEPT_IDS = "|".join(PARAMS["openalex_concepts"])
+INPUT_PATH = "data/openAlex/processed/"
+DATA_PATH_LOCAL = os.path.join(PROJECT_DIR, "inputs", "data")
+FIG_PATH = os.path.join(PROJECT_DIR, "outputs", "figures")
+MODEL_PATH = os.path.join(PROJECT_DIR, "outputs", "models")
```

Comment:
Perhaps you can create this paths as well if they don't exist? (eg, with `YOUR_PATH.mkdir(exist_ok=true parents=true)`) Otherwise, I'm getting errors when running for the first time